### ISSUE TITLE
DISABLED test_addmm_baddbmm_overflow_cuda_float16 (__main__.TestLinalgCUDA)

### ISSUE THREAD

**Issue by mruberry** [pytorch/pytorch#68125]
This test is failing consistently on _win-vs2019-cuda11.3-py3 / test (default, 2, 2, windows.8xlarge.nvidia.gpu) but apparently no other jobs. Failure snippet:

```
❌ Failure: TestLinalgCUDA.test_addmm_baddbmm_overflow_cuda_float16
Traceback (most recent call last):
  File "C:\actions-runner\_work\pytorch\pytorch\build\win_tmp\build\torch\testing\_internal\common_device_type.py", line 371, in instantiated_test
    result = test(self, **param_kwargs)
  File "C:\actions-runner\_work\pytorch\pytorch\build\win_tmp\build\torch\testing\_internal\common_device_type.py", line 891, in only_fn
    return fn(slf, *args, **kwargs)
  File "test_linalg.py", line 6139, in test_addmm_baddbmm_overflow
    self.assertTrue((out == 10000.).all())
AssertionError: tensor(False, device='cuda:0') is not true
```

Platforms: windows

cc @ezyang @gchanan @zou3519 @bdhirsh @jbschlosser @mruberry @jianyuh @nikitaved @pearu @walterddr @IvanYashchuk @xwang233 @Lezcano

**Comment by ngimel** [comment]
cc @eqy

**Comment by eqy** [comment]
Ah, looks like I neglected to add the switch here after #67946 .
Opened #68399

**Comment by mruberry** [comment]
Should be fixed by #68399
